### PR TITLE
Update from private docs site: Changes to What's new template tips

### PR DESCRIPTION
**Pull request by rhetoric101**
This is a PR based on changes that came over from a test of the private docs site. These are valid changes to our What's New? template style guide doc.

Reviewer: Here's what to look for: 

* We had a broken image showing a post of dark mode, but I couldn't find it and there was no images directory. I added an image of the Pixie announcement and added a caption.
* I inserted a Best Practices section at the bottom, mostly for writers.

### CODE REVIEW

File: src/content/docs/style-guide/article-templates/whats-new-style-guidelines.mdx [hunk 1]
```diff
@@ -137,4 +141,11 @@ Ordered list
 2. Step two
 3. Step three
 ```
+## Best practices
+
+Here are some tips to ensure that your posting goes smoothly:
 
+* **Publishing timing:** If you are a writer publishing a post from a contributor, pay attention to when the posts should be made public. You may need to stage the docs in preparation for a future release date.
+* **Links:** Be sure to use absolute links for all your posts, even for links that refer to the docs site.
+* **Filenames:** Don't include any spaces in your filenames. If you are a writer removing filename spaces from a contributor, be sure to ensure the name is unique so you avoid collisions with existing posts. One way to do this is to include the date in the filename.
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/article-templates/whats-new-style-guidelines.mdx:150]: "be sure to ensure" seems redundant to me. Maybe just "from a contributor, make sure the name..."?

File: src/content/docs/style-guide/article-templates/whats-new-style-guidelines.mdx [hunk 2]
```diff
@@ -137,4 +141,11 @@ Ordered list
...
+* **IDs:** For writers who publish posts, be sure to drop any changes to `whats-new-ids.json` before committing.
```
Comment (barbnewrelic) [inline src/content/docs/style-guide/article-templates/whats-new-style-guidelines.mdx:151]: I'm sorry, I don't quite understand what you mean by "drop any changes." Do you mean to "add new filenames, or update/delete existing filenames" ?

**Review by barbnewrelic — APPROVED** [review]
Just a few questions/comments, take 'em or leave 'em. But also, I noticed that this doc says to use sentence case. Our April 21, 2021 team meeting notes say to follow brand guidelines, which use title case not sentence case. Has something changed since then?

File: src/content/docs/style-guide/article-templates/whats-new-style-guidelines.mdx (hunk 1 above)
Comment (rhetoric101) [reply src/content/docs/style-guide/article-templates/whats-new-style-guidelines.mdx]: @barbnewrelic, thanks for catching that! I just pushed up a correction!

**Comment by rhetoric101** [comment]
@barbnewrelic, thanks for catching that old comment about sentence case in the title. Could you look at my revised callout? I wasn't able to get the anchor to work in Word, but the link the the editorial style guide does get the user in the general vicinity.

**Review by barbnewrelic — APPROVED** [review]
Looks great, thanks again for all your hard work on this!

**Review by barbnewrelic — APPROVED** [review]
Thanks for working so hard on this doc, great job!

**Pull request merged**